docs/reference-guides/rancher-security/rancher-security-best-practices.md

```diff
@@ -12,4 +12,10 @@ The upstream (local) Rancher instance provides information about the Rancher ver
 
 Adversaries can misuse this information to identify the running Rancher version and cross-relate it with potential bugs to exploit. If your upstream Rancher instance is publicly available on the web, use a Layer 7 firewall to block `/version` and `/rancherversion`.
 
-See [OWASP Web Application Security Testing - Enumerate Infrastructure and Application Admin Interfaces](https://owasp.org/www-project-web-security-testing-guide/stable/4-Web_Application_Security_Testing/02-Configuration_and_Deployment_Management_Testing/05-Enumerate_Infrastructure_and_Application_Admin_Interfaces.html) for more information on protecting your server.
\ No newline at end of file
+See [OWASP Web Application Security Testing - Enumerate Infrastructure and Application Admin Interfaces](https://owasp.org/www-project-web-security-testing-guide/stable/4-Web_Application_Security_Testing/02-Configuration_and_Deployment_Management_Testing/05-Enumerate_Infrastructure_and_Application_Admin_Interfaces.html) for more information on protecting your server.
+
+### Session management
+
+Some environments may require extra security controls regarding session management. The upstream Rancher instance does not implement a limit for active sessions for a specific user. 
```

Comment:
```suggestion
Some environments may require additional security controls regarding session management, for example, limiting the amount of concurrent active sessions for a given user or restricting where those sessions can be initiated from - to block specific geo-locations or impossible travel incidents. Such features are not supported out-of-the box in Rancher Manager. 
```

Reply:
I like this suggestion ^, would only suggest to simplify and remove the following part - `or impossible travel incidents`. It seems a bit too much for me and the geo-blocking mentioned earlier would already cover this IMO.